sscanss/editor/widgets.py

```diff
@@ -319,7 +319,7 @@ def __init__(self, parent):
         super().__init__(parent)
         self.parent = parent
 
-        self.instrument = self.parent.instrument
+        self.instrument = parent.instrument
```

Comment:
This should be `parent.presenter.model.instrument`. Please ensure all the widgets and dialogs are accessing the instrument correctly